www/components/pages/Home/Inbox/Inbox.js

```diff
@@ -23,17 +50,81 @@ export default function Inbox({ meetings, refresh }) {
     );
   });
 
-  if (lineItems.length) {
-    return <div className={styles.table}>{lineItems}</div>;
+  const handleNameChange = (event) => {
+    setFilters((prevState) => ({ ...prevState, name: event.target.value }));
+    setFetchingMeetings(true);
+  };
+
+  const handleOwnersChange = (event, value) => {
+    setFilters((prevState) => ({ ...prevState, owners: value.owner_id }));
+  };
+
+  const users = Object.values(
+    [
+      ...new Set(
+        meetings.map((item) => ({
+          name: item.owner.email,
+          owner_id: item.owner._id,
+        }))
+      ),
```

Comment:
We don't need to use the `Set` here. Set's only work for primitive types in js. This is because the variable's values for the objects passed into a Set are just pointers to memory addresses where the object's value is stored, not the value of the object. So since each object has its own memory address and pointer, the Set will think they are all unique.